docs/cli/kops_create_secret.md

```diff
@@ -52,6 +52,7 @@ Create a secret
 ### SEE ALSO
 
 * [kops create](kops_create.md)	 - Create a resource by command line, filename or stdin.
+* [kops create secret ciliumpassword](kops_create_secret_ciliumpassword.md)	 - Create a cilium encryption config.
```

Comment:
Nit, but I think "encryption key" is more correct than "encryption config"

Reply:
Agreed, fixing!

Reply:
@olemarkus - Fixed and pushed. Looks like that caused your `/lgtm` to be discarded.
Would you mind re-adding that if it looks okay to you?
Thanks!